setuptools_odoo/make_default_setup.py

```diff
@@ -120,6 +121,55 @@ def make_default_setup_addons_dir(addons_dir, force,
                                  odoo_version_override)
 
 
+def clean_setup_addons_dir(addons_dir):
+    paths_to_remove = []
+
+    for addon_name in os.listdir(addons_dir):
+        addon_dir = os.path.join(addons_dir, addon_name)
+        is_installable = is_installable_addon(addon_dir)
+        try:
+            # File or directory is not an addon. Skip it
+            manifest = read_manifest(addon_dir)
+        except NoManifestFound:
+            continue
+
+        addon_setup_dir = os.path.join(addons_dir, 'setup', addon_name)
+        if not is_installable:
+            paths_to_remove.append(addon_setup_dir)
+            continue
+        version, odoo_version_info = _get_version(
+            addon_dir, manifest, git_post_version=False)
+        odoo_version = int(version.split('.')[0])
+
+        if odoo_version >= 10:
+            paths_to_remove.append(
+                os.path.join(addon_setup_dir, 'odoo_addons'))
+        if odoo_version >= 11:
+            paths_to_remove.append(
+                os.path.join(addon_setup_dir, 'odoo', '__init__.py'))
+            paths_to_remove.append(
+                os.path.join(addon_setup_dir, 'odoo', 'addons', '__init__.py'))
+
+    for path_to_remove in paths_to_remove:
+        path = os.path.abspath(path_to_remove)
+        if os.path.exists(path):
+            subprocess.check_call([
+                'git', 'rm', '-rf', path,
+            ])
+
+
+def make_default_setup_commit_files(addons_dir):
+    subprocess.check_call(['git', 'add', 'setup'], cwd=addons_dir)
+    commit_needed = subprocess.call([
+        'git', 'diff', '--quiet', '--cached',
+        '--exit-code', 'setup'
+    ], cwd=addons_dir) != 0
```

Comment:
cwd=addons_dir/'setup'?

Reply:
The ```setup``` directory is provided in the above call.